lib/paperclip/storage/google_drive.rb

```diff
@@ -156,7 +156,7 @@ def public_url_for title
       def search_for_title(title)
         parameters = {
                 'folderId' => find_public_folder,
-                'q' => "title contains '#{title}'", # full_title
```

Comment:
I think it'd be good to provide the option for different operators via an option (with the default as `contains`). I'm sure there could be plenty of use cases that would need one operator or another (such as `=`). This way it could be introduced without breaking any existing projects?